src/nvim/main.c

```diff
@@ -672,6 +673,8 @@ void getout(int exitval)
     garbage_collect(false);
   }
 
+  free_ctx_stack();
```

Comment:
normally done in `free_all_mem()`. I moved it there. Also renamed to `ctx_free_all`.

(Typically there's no reason to explicitly free anything on exit, instead let the OS do it. That's why `mch_exit` is like this:

```c
#ifdef EXITFREE
  free_all_mem()
#endif
```